src/test/resources/com/microsoft/azure/vmagent/test/jcasc/advanced.yaml

```diff
@@ -0,0 +1,55 @@
+jenkins:
```

Comment:
@gavinfish here's an example of how it would look for a user configuring it with the current databinding names